### ISSUE TITLE
Missing Direct Play profile for custom Chromium build since Jellyfin 10.6.x makes it impossible to Direct Play HEVC files

### ISSUE THREAD

**Issue by whalehub** [jellyfin/jellyfin#3900]
**Media Info of the file**

<details>
  <summary>Spoiler</summary>
<p></p>

```
General
Unique ID                                : <redacted>
Complete name                            : /media/anime/[C] The Money of Soul and Possibility Control (2011)/Season 01/[C] The Money of Soul and Possibility Control (2011) - S01E01 - Complication.mkv
Format                                   : Matroska
Format version                           : Version 4
File size                                : 388 MiB
Duration                                 : 22 min 52 s
Overall bit rate                         : 2 371 kb/s
Movie name                               : <redacted>
Encoded date                             : UTC 2017-07-16 10:47:07
Writing application                      : mkvmerge v13.0.0 ('The Juggler') 64bit
Writing library                          : libebml v1.3.4 + libmatroska v1.4.5
Attachments                              : ARIAL.TTF / ARLRDBD.TTF / Artois.ttf / AVGARDNEclipse.ttf / kaiu.ttf / segoepr.ttf / Vesta-Bold.otf

Video
ID                                       : 1
Format                                   : HEVC
Format/Info                              : High Efficiency Video Coding
Format profile                           : Main 10@L5@Main
Codec ID                                 : V_MPEGH/ISO/HEVC
Duration                                 : 22 min 52 s
Bit rate                                 : 1 990 kb/s
Width                                    : 1 920 pixels
Height                                   : 1 080 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Constant
Frame rate                               : 23.976 (24000/1001) FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 10 bits
Bits/(Pixel*Frame)                       : 0.040
Stream size                              : 325 MiB (84%)
Title                                    : <redacted>
Writing library                          : x265 2.4+14-bc0e9bd7c08f5ddc:[Windows][MSVC 1800][64 bit] 10bit: KG7x [x265.ru]
Encoding settings                        : cpuid=1173503 / frame-threads=2 / numa-pools=4 / wpp / no-pmode / no-pme / no-psnr / no-ssim / log-level=2 / input-csp=1 / input-res=1920x1080 / interlace=0 / total-frames=32896 / level-idc=0 / high-tier=1 / uhd-bd=0 / ref=6 / no-allow-non-conformance / no-repeat-headers / annexb / no-aud / no-hrd / info / hash=0 / no-temporal-layers / open-gop / min-keyint=23 / keyint=250 / bframes=8 / b-adapt=2 / b-pyramid / bframe-bias=0 / rc-lookahead=60 / lookahead-slices=0 / scenecut=40 / no-intra-refresh / ctu=32 / min-cu-size=8 / rect / no-amp / max-tu-size=16 / tu-inter-depth=3 / tu-intra-depth=3 / limit-tu=4 / rdoq-level=2 / dynamic-rd=0.00 / no-ssim-rd / signhide / no-tskip / nr-intra=0 / nr-inter=0 / no-constrained-intra / no-strong-intra-smoothing / max-merge=4 / limit-refs=3 / limit-modes / me=3 / subme=5 / merange=57 / temporal-mvp / weightp / weightb / no-analyze-src-pics / no-deblock / no-sao / no-sao-non-deblock / rd=5 / no-early-skip / rskip / no-fast-intra / no-tskip-fast / no-cu-lossless / b-intra / rdpenalty=0 / psy-rd=0.80 / psy-rdoq=2.00 / no-rd-refine / analysis-mode=0 / no-lossless / cbqpoffs=0 / crqpoffs=0 / rc=crf / crf=22.0 / qcomp=0.60 / qpstep=4 / stats-write=0 / stats-read=0 / ipratio=1.40 / pbratio=1.30 / aq-mode=2 / aq-strength=1.00 / cutree / zone-count=0 / no-strict-cbr / qg-size=32 / no-rc-grain / qpmax=69 / qpmin=0 / sar=0 / overscan=0 / videoformat=5 / range=0 / colorprim=2 / transfer=2 / colormatrix=2 / chromaloc=0 / display-window=0 / max-cll=0,0 / min-luma=0 / max-luma=1023 / log2-max-poc-lsb=8 / vui-timing-info / vui-hrd-info / slices=1 / opt-qp-pps / opt-ref-list-length-pps / no-multi-pass-opt-rps / scenecut-bias=0.05 / no-opt-cu-delta-qp / no-aq-motion / no-hdr / no-hdr-opt / no-dhdr10-opt / refine-level=5 / no-limit-sao / ctu-info=0
Default                                  : Yes
Forced                                   : No

Audio #1
ID                                       : 2
Format                                   : AAC LC
Format/Info                              : Advanced Audio Codec Low Complexity
Codec ID                                 : A_AAC-2
Duration                                 : 22 min 52 s
Bit rate                                 : 239 kb/s
Channel(s)                               : 6 channels
Channel layout                           : C L R Ls Rs LFE
Sampling rate                            : 48.0 kHz
Frame rate                               : 46.875 FPS (1024 SPF)
Compression mode                         : Lossy
Delay relative to video                  : 9 ms
Stream size                              : 39.2 MiB (10%)
Language                                 : English
Default                                  : No
Forced                                   : No

Audio #2
ID                                       : 3
Format                                   : AAC LC
Format/Info                              : Advanced Audio Codec Low Complexity
Codec ID                                 : A_AAC-2
Duration                                 : 22 min 52 s
Bit rate                                 : 92.9 kb/s
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 48.0 kHz
Frame rate                               : 46.875 FPS (1024 SPF)
Compression mode                         : Lossy
Delay relative to video                  : 9 ms
Stream size                              : 15.2 MiB (4%)
Language                                 : Japanese
Default                                  : No
Forced                                   : No

Text #1
ID                                       : 4
Format                                   : ASS
Codec ID                                 : S_TEXT/ASS
Codec ID/Info                            : Advanced Sub Station Alpha
Duration                                 : 22 min 44 s
Bit rate                                 : 47 b/s
Count of elements                        : 82
Compression mode                         : Lossless
Stream size                              : 7.98 KiB (0%)
Title                                    : Signs [Doki]
Language                                 : English
Default                                  : Yes
Forced                                   : Yes

Text #2
ID                                       : 5
Format                                   : ASS
Codec ID                                 : S_TEXT/ASS
Codec ID/Info                            : Advanced Sub Station Alpha
Duration                                 : 22 min 44 s
Bit rate                                 : 173 b/s
Count of elements                        : 385
Compression mode                         : Lossless
Stream size                              : 28.9 KiB (0%)
Title                                    : Subtitles [Doki]
Language                                 : English
Default                                  : No
Forced                                   : No

Text #3
ID                                       : 6
Format                                   : VobSub
Muxing mode                              : zlib
Codec ID                                 : S_VOBSUB
Codec ID/Info                            : Picture based subtitle format used on DVDs
Duration                                 : 22 min 40 s
Bit rate                                 : 12.5 kb/s
Count of elements                        : 394
Stream size                              : 2.02 MiB (1%)
Title                                    : English Subs Retail
Language                                 : English
Default                                  : No
Forced                                   : No

Menu
00:00:00.000                             : en:Prologue
00:08:13.952                             : en:OP
00:09:44.000                             : en:Part A
00:14:11.017                             : en:Part B
00:21:21.071                             : en:ED
```
</details>

**Logs**

<details>
  <summary>Spoiler</summary>
<p></p>

```
[2020-08-14 18:46:24.255 +02:00] [INF] [19] MediaBrowser.Api.Playback.MediaInfoService: Profile: "Unknown Profile", Path: "/media/anime/[C] The Money of Soul and Possibility Control (2011)/Season 01/[C] The Money of Soul and Possibility Control (2011) - S01E01 - Complication.mkv", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2020-08-14 18:46:24.255 +02:00] [INF] [19] MediaBrowser.Api.Playback.MediaInfoService: Profile: "Unknown Profile", No direct play profiles found for Path: "/media/anime/[C] The Money of Soul and Possibility Control (2011)/Season 01/[C] The Money of Soul and Possibility Control (2011) - S01E01 - Complication.mkv"
```
</details>

**FFmpeg Logs**

<details>
  <summary>Spoiler</summary>
<p></p>

```
http://jellyfin.datahoarder.dev/videos/eb44aaa7-b65f-03a0-f876-e23713922fff/hls1/main/0.ts?DeviceId=TW96aWxsYS81LjAgKFgxMTsgTGludXggeDg2XzY0KSBBcHBsZVdlYktpdC81MzcuMzYgKEtIVE1MLCBsaWtlIEdlY2tvKSBDaHJvbWUvODQuMC40MTQ3LjEyNSBTYWZhcmkvNTM3LjM2fDE1OTc0MjM1Mjc2ODY1&MediaSourceId=eb44aaa7b65f03a0f876e23713922fff&VideoCodec=h264&AudioCodec=mp3,aac&AudioStreamIndex=1&VideoBitrate=139680000&AudioBitrate=320000&PlaySessionId=b18c7ff7d78c4b4b8d0abcca33b12673&api_key=fce5bacca7ca4823914f436b19ab4f26&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=42932597890f67891d0e2ab1e2817f96&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&h264-profile=high,main,baseline,constrainedbaseline,high10&h264-level=51&h264-deinterlace=true&TranscodeReasons=VideoCodecNotSupported

{"Protocol":"File","Id":"eb44aaa7b65f03a0f876e23713922fff","Path":"/media/anime/[C] The Money of Soul and Possibility Control (2011)/Season 01/[C] The Money of Soul and Possibility Control (2011) - S01E01 - Complication.mkv","Type":"Default","Container":"mkv,webm","Size":406577568,"Name":"/media/anime/[C] The Money of Soul and Possibility Control (2011)/Season 01/[C] The Money of Soul and Possibility Control (2011) - S01E01 - Complication","IsRemote":false,"ETag":"42932597890f67891d0e2ab1e2817f96","RunTimeTicks":13720410112,"ReadAtNativeFramerate":false,"IgnoreDts":false,"IgnoreIndex":false,"GenPtsInput":false,"SupportsTranscoding":true,"SupportsDirectStream":true,"SupportsDirectPlay":true,"IsInfiniteStream":false,"RequiresOpening":false,"RequiresClosing":false,"RequiresLooping":false,"SupportsProbing":true,"VideoType":"VideoFile","MediaStreams":[{"Codec":"hevc","TimeBase":"1/1000","CodecTimeBase":"1001/24000","Title":"<redacted>","VideoRange":"SDR","DisplayTitle":"1080p HEVC","IsInterlaced":false,"BitRate":2370643,"RefFrames":1,"IsDefault":true,"IsForced":false,"Height":1080,"Width":1920,"AverageFrameRate":23.976025,"RealFrameRate":23.976025,"Profile":"Main 10","Type":"Video","AspectRatio":"16:9","Index":0,"IsExternal":false,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"PixelFormat":"yuv420p10le","Level":150},{"Codec":"aac","Language":"eng","TimeBase":"1/1000","CodecTimeBase":"1/48000","DisplayTitle":"Eng AAC 5.1","IsInterlaced":false,"ChannelLayout":"5.1","BitRate":320000,"Channels":6,"SampleRate":48000,"IsDefault":false,"IsForced":false,"Profile":"LC","Type":"Audio","Index":1,"IsExternal":false,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Level":0},{"Codec":"aac","Language":"jpn","TimeBase":"1/1000","CodecTimeBase":"1/48000","DisplayTitle":"Jpn AAC stereo","IsInterlaced":false,"ChannelLayout":"stereo","BitRate":192000,"Channels":2,"SampleRate":48000,"IsDefault":false,"IsForced":false,"Profile":"LC","Type":"Audio","Index":2,"IsExternal":false,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Level":0},{"Codec":"ass","Language":"eng","TimeBase":"1/1000","CodecTimeBase":"0/1","Title":"Signs [Doki]","localizedUndefined":"Undefined","localizedDefault":"Default","localizedForced":"Forced","DisplayTitle":"Signs [Doki] - Eng - Default - Forced","IsInterlaced":false,"IsDefault":true,"IsForced":true,"Type":"Subtitle","Index":3,"IsExternal":false,"IsTextSubtitleStream":true,"SupportsExternalStream":true,"Level":0},{"Codec":"ass","Language":"eng","TimeBase":"1/1000","CodecTimeBase":"0/1","Title":"Subtitles [Doki]","localizedUndefined":"Undefined","localizedDefault":"Default","localizedForced":"Forced","DisplayTitle":"Subtitles [Doki] - Eng","IsInterlaced":false,"IsDefault":false,"IsForced":false,"Type":"Subtitle","Index":4,"IsExternal":false,"IsTextSubtitleStream":true,"SupportsExternalStream":true,"Level":0},{"Codec":"DVDSUB","Language":"eng","TimeBase":"1/1000","CodecTimeBase":"0/1","Title":"English Subs Retail","localizedUndefined":"Undefined","localizedDefault":"Default","localizedForced":"Forced","DisplayTitle":"English Subs Retail","IsInterlaced":false,"IsDefault":false,"IsForced":false,"Type":"Subtitle","Index":5,"IsExternal":false,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Level":0}],"MediaAttachments":[{"Codec":"ttf","CodecTag":"[0][0][0][0]","Index":6,"FileName":"ARIAL.TTF","MimeType":"application/x-truetype-font"},{"Codec":"ttf","CodecTag":"[0][0][0][0]","Index":7,"FileName":"ARLRDBD.TTF","MimeType":"application/x-truetype-font"},{"Codec":"ttf","CodecTag":"[0][0][0][0]","Index":8,"FileName":"Artois.ttf","MimeType":"application/x-truetype-font"},{"Codec":"ttf","CodecTag":"[0][0][0][0]","Index":9,"FileName":"AVGARDNEclipse.ttf","MimeType":"application/x-truetype-font"},{"Codec":"ttf","CodecTag":"[0][0][0][0]","Index":10,"FileName":"kaiu.ttf","MimeType":"application/x-truetype-font"},{"Codec":"ttf","CodecTag":"[0][0][0][0]","Index":11,"FileName":"segoepr.ttf","MimeType":"application/x-truetype-font"},{"Codec":"ttf","CodecTag":"[0][0][0][0]","Index":12,"FileName":"Vesta-Bold.otf","MimeType":"application/x-truetype-font"}],"Formats":[],"Bitrate":2882643,"RequiredHttpHeaders":{}}

/usr/lib/jellyfin-ffmpeg/ffmpeg -f matroska,webm -i file:"/media/anime/[C] The Money of Soul and Possibility Control (2011)/Season 01/[C] The Money of Soul and Possibility Control (2011) - S01E01 - Complication.mkv" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:0 -map 0:1 -map -0:s -codec:v:0 libx264 -pix_fmt yuv420p -preset veryfast -crf 23 -maxrate 9482572 -bufsize 18965144 -profile:v high -level 4.1 -x264opts:0 subme=0:me_range=4:rc_lookahead=10:me=dia:no_chroma_me:8x8dct=0:partitions=none  -force_key_frames:0 "expr:gte(t,0+n_forced*3)" -g 72 -keyint_min 72 -sc_threshold 0 -vf "scale=trunc(min(max(iw\,ih*dar)\,1920)/2)*2:trunc(ow/dar/2)*2" -start_at_zero -vsync -1 -codec:a:0 libmp3lame -ac 2 -ab 320000 -af "volume=2" -copyts -avoid_negative_ts disabled -f hls -max_delay 5000000 -hls_time 3 -individual_header_trailer 0 -hls_segment_type mpegts -start_number 0 -hls_segment_filename "/config/data/transcodes/84bcfc2daf0b51e27b750996d9ab52e2%d.ts" -hls_playlist_type vod -hls_list_size 0 -y "/config/data/transcodes/84bcfc2daf0b51e27b750996d9ab52e2.m3u8"


ffmpeg version 4.3.1 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 7 (Ubuntu 7.5.0-3ubuntu1~18.04)
  configuration: --prefix=/usr/lib/jellyfin-ffmpeg --target-os=linux --disable-doc --disable-ffplay --disable-shared --disable-libxcb --disable-sdl2 --disable-xlib --enable-gpl --enable-version3 --enable-static --enable-libfontconfig --enable-fontconfig --enable-gmp --enable-gnutls --enable-libass --enable-libbluray --enable-libdrm --enable-libfreetype --enable-libfribidi --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libwebp --enable-libx264 --enable-libx265 --enable-libzvbi --arch=amd64 --enable-libzimg --enable-opencl --enable-vaapi --enable-amf --enable-libmfx --enable-vdpau --enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-nvdec --enable-ffnvcodec --enable-libdav1d
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
  libpostproc    55.  7.100 / 55.  7.100
Input #0, matroska,webm, from 'file:/media/anime/[C] The Money of Soul and Possibility Control (2011)/Season 01/[C] The Money of Soul and Possibility Control (2011) - S01E01 - Complication.mkv':
  Metadata:
    title           : <redacted>
    encoder         : libebml v1.3.4 + libmatroska v1.4.5
    creation_time   : 2017-07-16T10:47:07.000000Z
  Duration: 00:22:52.04, start: 0.000000, bitrate: 2370 kb/s
    Chapter #0:0: start 0.000000, end 493.952000
    Metadata:
      title           : Prologue
    Chapter #0:1: start 493.952000, end 584.000000
    Metadata:
      title           : OP
    Chapter #0:2: start 584.000000, end 851.017000
    Metadata:
      title           : Part A
    Chapter #0:3: start 851.017000, end 1281.071000
    Metadata:
      title           : Part B
    Chapter #0:4: start 1281.071000, end 1372.041000
    Metadata:
      title           : ED
    Stream #0:0: Video: hevc (Main 10), yuv420p10le(tv), 1920x1080, SAR 1:1 DAR 16:9, 23.98 fps, 23.98 tbr, 1k tbn, 23.98 tbc (default)
    Metadata:
      title           : <redacted>
      BPS             : 1990021
      BPS-eng         : 1990021
      DURATION        : 00:22:52.038000000
      DURATION-eng    : 00:22:52.038000000
      NUMBER_OF_FRAMES: 32896
      NUMBER_OF_FRAMES-eng: 32896
      NUMBER_OF_BYTES : 341298192
      NUMBER_OF_BYTES-eng: 341298192
      _STATISTICS_WRITING_APP: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_APP-eng: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_DATE_UTC: 2017-07-16 10:47:07
      _STATISTICS_WRITING_DATE_UTC-eng: 2017-07-16 10:47:07
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
    Stream #0:1(eng): Audio: aac (LC), 48000 Hz, 5.1, fltp
    Metadata:
      BPS             : 239473
      BPS-eng         : 239473
      DURATION        : 00:22:52.032000000
      DURATION-eng    : 00:22:52.032000000
      NUMBER_OF_FRAMES: 64314
      NUMBER_OF_FRAMES-eng: 64314
      NUMBER_OF_BYTES : 41070631
      NUMBER_OF_BYTES-eng: 41070631
      _STATISTICS_WRITING_APP: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_APP-eng: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_DATE_UTC: 2017-07-16 10:47:07
      _STATISTICS_WRITING_DATE_UTC-eng: 2017-07-16 10:47:07
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
    Stream #0:2(jpn): Audio: aac (LC), 48000 Hz, stereo, fltp
    Metadata:
      BPS             : 92876
      BPS-eng         : 92876
      DURATION        : 00:22:52.032000000
      DURATION-eng    : 00:22:52.032000000
      NUMBER_OF_FRAMES: 64314
      NUMBER_OF_FRAMES-eng: 64314
      NUMBER_OF_BYTES : 15928759
      NUMBER_OF_BYTES-eng: 15928759
      _STATISTICS_WRITING_APP: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_APP-eng: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_DATE_UTC: 2017-07-16 10:47:07
      _STATISTICS_WRITING_DATE_UTC-eng: 2017-07-16 10:47:07
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
    Stream #0:3(eng): Subtitle: ass (default) (forced)
    Metadata:
      title           : Signs [Doki]
      BPS             : 47
      BPS-eng         : 47
      DURATION        : 00:22:44.280000000
      DURATION-eng    : 00:22:44.280000000
      NUMBER_OF_FRAMES: 82
      NUMBER_OF_FRAMES-eng: 82
      NUMBER_OF_BYTES : 8176
      NUMBER_OF_BYTES-eng: 8176
      _STATISTICS_WRITING_APP: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_APP-eng: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_DATE_UTC: 2017-07-16 10:47:07
      _STATISTICS_WRITING_DATE_UTC-eng: 2017-07-16 10:47:07
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
    Stream #0:4(eng): Subtitle: ass
    Metadata:
      title           : Subtitles [Doki]
      BPS             : 173
      BPS-eng         : 173
      DURATION        : 00:22:44.280000000
      DURATION-eng    : 00:22:44.280000000
      NUMBER_OF_FRAMES: 385
      NUMBER_OF_FRAMES-eng: 385
      NUMBER_OF_BYTES : 29561
      NUMBER_OF_BYTES-eng: 29561
      _STATISTICS_WRITING_APP: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_APP-eng: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_DATE_UTC: 2017-07-16 10:47:07
      _STATISTICS_WRITING_DATE_UTC-eng: 2017-07-16 10:47:07
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
    Stream #0:5(eng): Subtitle: dvd_subtitle, 1920x1080
    Metadata:
      title           : English Subs Retail
      BPS             : 12464
      BPS-eng         : 12464
      DURATION        : 00:22:40.892000000
      DURATION-eng    : 00:22:40.892000000
      NUMBER_OF_FRAMES: 394
      NUMBER_OF_FRAMES-eng: 394
      NUMBER_OF_BYTES : 2120369
      NUMBER_OF_BYTES-eng: 2120369
      _STATISTICS_WRITING_APP: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_APP-eng: mkvmerge v13.0.0 ('The Juggler') 64bit
      _STATISTICS_WRITING_DATE_UTC: 2017-07-16 10:47:07
      _STATISTICS_WRITING_DATE_UTC-eng: 2017-07-16 10:47:07
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
      _STATISTICS_TAGS-eng: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
    Stream #0:6: Attachment: ttf
    Metadata:
      filename        : ARIAL.TTF
      mimetype        : application/x-truetype-font
    Stream #0:7: Attachment: ttf
    Metadata:
      filename        : ARLRDBD.TTF
      mimetype        : application/x-truetype-font
    Stream #0:8: Attachment: ttf
    Metadata:
      filename        : Artois.ttf
      mimetype        : application/x-truetype-font
    Stream #0:9: Attachment: ttf
    Metadata:
      filename        : AVGARDNEclipse.ttf
      mimetype        : application/x-truetype-font
    Stream #0:10: Attachment: ttf
    Metadata:
      filename        : kaiu.ttf
      mimetype        : application/x-truetype-font
    Stream #0:11: Attachment: ttf
    Metadata:
      filename        : segoepr.ttf
      mimetype        : application/x-truetype-font
    Stream #0:12: Attachment: ttf
    Metadata:
      filename        : Vesta-Bold.otf
      mimetype        : application/x-truetype-font
Stream mapping:
  Stream #0:0 -> #0:0 (hevc (native) -> h264 (libx264))
  Stream #0:1 -> #0:1 (aac (native) -> mp3 (libmp3lame))
Press [q] to stop, [?] for help
[libx264 @ 0x56197de92180] using SAR=1/1
[libx264 @ 0x56197de92180] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX
[libx264 @ 0x56197de92180] profile Main, level 4.1
[libx264 @ 0x56197de92180] 264 - core 152 r2854 e9a5903 - H.264/MPEG-4 AVC codec - Copyleft 2003-2017 - http://www.videolan.org/x264.html - options: cabac=1 ref=1 deblock=1:0:0 analyse=0x1:0 me=dia subme=0 psy=1 psy_rd=1.00:0.00 mixed_ref=0 me_range=4 chroma_me=0 trellis=0 8x8dct=0 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=0 threads=34 lookahead_threads=5 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_adapt=1 b_bias=0 direct=1 weightb=1 open_gop=0 weightp=1 keyint=72 keyint_min=37 scenecut=0 intra_refresh=0 rc_lookahead=10 rc=crf mbtree=1 crf=23.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 vbv_maxrate=9482 vbv_bufsize=18965 crf_max=0.0 nal_hrd=none filler=0 ip_ratio=1.40 aq=1:1.00
Output #0, hls, to '/config/data/transcodes/84bcfc2daf0b51e27b750996d9ab52e2.m3u8':
  Metadata:
    encoder         : Lavf58.45.100
    Stream #0:0: Video: h264 (libx264), yuv420p, 1920x1080 [SAR 1:1 DAR 16:9], q=-1--1, 23.98 fps, 90k tbn, 23.98 tbc (default)
    Metadata:
      encoder         : Lavc58.91.100 libx264
    Side data:
      cpb: bitrate max/min/avg: 9482000/0/0 buffer size: 18965000 vbv_delay: N/A
    Stream #0:1: Audio: mp3 (libmp3lame), 48000 Hz, stereo, fltp, 320 kb/s
    Metadata:
      encoder         : Lavc58.91.100 libmp3lame
frame=    2 fps=0.4 q=0.0 size=N/A time=00:00:00.77 bitrate=N/A speed=0.167x    
frame=    6 fps=1.1 q=0.0 size=N/A time=00:00:00.92 bitrate=N/A speed=0.176x    
frame=   12 fps=2.1 q=0.0 size=N/A time=00:00:01.28 bitrate=N/A speed=0.223x    
frame=   18 fps=2.9 q=0.0 size=N/A time=00:00:01.61 bitrate=N/A speed=0.256x    
frame=   24 fps=3.4 q=0.0 size=N/A time=00:00:01.61 bitrate=N/A speed=0.231x    
frame=   30 fps=3.9 q=0.0 size=N/A time=00:00:01.95 bitrate=N/A speed=0.256x    
frame=   33 fps=4.1 q=0.0 size=N/A time=00:00:02.12 bitrate=N/A speed=0.26x    
frame=   38 fps=4.2 q=0.0 size=N/A time=00:00:02.45 bitrate=N/A speed=0.272x    
frame=   40 fps=4.0 q=0.0 size=N/A time=00:00:02.45 bitrate=N/A speed=0.243x    
frame=   42 fps=3.9 q=0.0 size=N/A time=00:00:02.45 bitrate=N/A speed=0.229x    
frame=   45 fps=4.0 q=0.0 size=N/A time=00:00:02.62 bitrate=N/A speed=0.234x    
frame=   56 fps=4.7 q=28.0 size=N/A time=00:00:03.15 bitrate=N/A speed=0.263x    
frame=  114 fps=9.1 q=28.0 size=N/A time=00:00:05.62 bitrate=N/A speed=0.45x    
[hls @ 0x56197def7840] Opening '/config/data/transcodes/84bcfc2daf0b51e27b750996d9ab52e20.ts' for writing
frame=  174 fps= 13 q=28.0 size=N/A time=00:00:08.00 bitrate=N/A speed=0.616x    
[hls @ 0x56197def7840] Opening '/config/data/transcodes/84bcfc2daf0b51e27b750996d9ab52e21.ts' for writing
frame=  232 fps= 17 q=25.0 size=N/A time=00:00:10.28 bitrate=N/A speed=0.761x    
[hls @ 0x56197def7840] Opening '/config/data/transcodes/84bcfc2daf0b51e27b750996d9ab52e22.ts' for writing
frame=  281 fps= 20 q=28.0 size=N/A time=00:00:12.39 bitrate=N/A speed=0.884x    
frame=  331 fps= 23 q=28.0 size=N/A time=00:00:14.38 bitrate=N/A speed=0.99x    
[hls @ 0x56197def7840] Opening '/config/data/transcodes/84bcfc2daf0b51e27b750996d9ab52e23.ts' for writing
frame=  384 fps= 26 q=28.0 size=N/A time=00:00:16.83 bitrate=N/A speed=1.12x    
[hls @ 0x56197def7840] Opening '/config/data/transcodes/84bcfc2daf0b51e27b750996d9ab52e24.ts' for writing
frame=  435 fps= 28 q=28.0 size=N/A time=00:00:18.94 bitrate=N/A speed=1.22x    
[hls @ 0x56197def7840] Opening '/config/data/transcodes/84bcfc2daf0b51e27b750996d9ab52e25.ts' for writing
[hls @ 0x56197def7840] Opening '/config/data/transcodes/84bcfc2daf0b51e27b750996d9ab52e26.ts' for writing
frame=  471 fps= 30 q=-1.0 Lsize=N/A time=00:00:20.52 bitrate=N/A speed=1.29x    
video:1469kB audio:802kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
[libx264 @ 0x56197de92180] frame I:7     Avg QP:16.45  size:130232
[libx264 @ 0x56197de92180] frame P:118   Avg QP:17.62  size:  4730
[libx264 @ 0x56197de92180] frame B:346   Avg QP:22.82  size:    99
[libx264 @ 0x56197de92180] consecutive B-frames:  1.5%  0.4%  3.8% 94.3%
[libx264 @ 0x56197de92180] mb I  I16..4: 36.2%  0.0% 63.8%
[libx264 @ 0x56197de92180] mb P  I16..4:  4.8%  0.0%  0.0%  P16..4:  8.2%  0.0%  0.0%  0.0%  0.0%    skip:87.0%
[libx264 @ 0x56197de92180] mb B  I16..4:  0.0%  0.0%  0.0%  B16..8:  0.0%  0.0%  0.0%  direct: 0.3%  skip:99.7%  L0: 7.0% L1:93.0% BI: 0.0%
[libx264 @ 0x56197de92180] coded y,uvDC,uvAC intra: 55.4% 48.4% 25.1% inter: 0.3% 1.8% 0.0%
[libx264 @ 0x56197de92180] i16 v,h,dc,p: 46% 26% 10% 18%
[libx264 @ 0x56197de92180] i4 v,h,dc,ddl,ddr,vr,hd,vl,hu: 31% 29% 10%  6%  5%  4%  5%  4%  6%
[libx264 @ 0x56197de92180] i8c dc,h,v,p: 47% 23% 23%  6%
[libx264 @ 0x56197de92180] Weighted P-Frames: Y:1.7% UV:1.7%
[libx264 @ 0x56197de92180] kb/s:612.48
```
</details>

**Server System (please complete the following information):**
 - **OS:** Docker on Debian 10
 - **Jellyfin Version:** 10.6.2
 - **Hardware settings & device:** No hardware acceleration, 2x Xeon E5-2697v2
 - **Reverse proxy:** Nginx
 - **Other hardware notes:** N/A
 
**Client System (please complete the following information):**
 - **Device:** Desktop PC
 - **OS:** Debian 10
 - **Client:** Web/Browser
 - **Browser (if Web client):** Custom Chromium build from [Woolyss](https://chromium.woolyss.com/#linux) [all-codecs+]
 - **Client and Browser Version:** 10.6.2 and 84.0.4147.125

**Additional Info**
I was able to Direct Play HEVC files using this custom Chromium build prior to upgrading to Jellyfin 10.6.x.

**Edit:** The same file can be Direct Played with this custom Chromium build in the current stable version of Emby.

**Comment by telans** [comment]
This should probably be transfered to jellyfin/jellyfin-web as it deals with [browserDeviceProfile](https://github.com/jellyfin/jellyfin-web/blob/master/src/scripts/browserDeviceProfile.js) I believe

**Comment by whalehub** [comment]
Bump

**Comment by whalehub** [comment]
Closed in favor of #2588.
